packages/babel-plugin-ember-test-metadata/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "babel-plugin-ember-test-metadata",
+  "version": "1.2.3",
+  "description": "A Babel plugin for Ember apps that adds a test's file path to test metadata",
+  "keywords": [],
+  "repository": "https://github.com/babel-plugin-ember-test-metadata/babel-plugin-ember-test-metadata",
+  "license": "MIT",
+  "author": {
+    "name": "Glenn Cueto",
+    "email": "gcueto@linkedin.com"
+  },
+  "main": "dist/index.js",
+  "files": [
+    "dist/*.js"
+  ],
+  "scripts": {
+    "prepublishOnly": "npm run build",
+    "prebuild": "rm -rf dist",
+    "build": "babel src --out-dir dist --copy-files --ignore **/*.test.js",
+    "postbuild": "rm -rf dist/**/*.test.js",
```

Comment:
this feels odd... do we need to transpile tests into dist to begin with?

Reply:
This isn't any different than what we did before. If we want to change this infra, let's keep this separate from the monorepo changes. Happy to look at this again after.